### PR TITLE
Resolves #2001

### LINKED ISSUE #2001: Invalid terminology description KCC
This page mentions 'Kerberos consistency checker (KCC)' a few times. This should be "Knowledge Consistency Checker (KCC)".

Rgds,
Eric

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 415995a5-2adb-fdb4-45d4-455d52604d83
* Version Independent ID: 191e4311-10e1-50b7-8926-06bfb8a4fdd6
* Content: [New-ADReplicationSite (addsadministration)](https://docs.microsoft.com/en-us/powershell/module/addsadministration/new-adreplicationsite?view=win10-ps)
* Content Source: [docset/windows/addsadministration/New-ADReplicationSite.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/addsadministration/New-ADReplicationSite.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Typo in KCC definition.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2001

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md [hunk 1]
```diff
@@ -245,30 +217,16 @@ Accept wildcard characters: False
...
-GUID (objectGUID)
-
-Example: 599c3d2e-f72d-4d20-8a88-030d99495f20
-
-Security Identifier (objectSid)
-
-Example: S-1-5-21-3165297888-301567370-576410423-1103
-
-SAM Account Name (sAMAccountName)
+Specifies the user or group that manages the object by providing one of the following property values :
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md:220]:
```suggestion
Specifies the user or group that manages the object by providing one of the following property values: 
```

File: docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md [hunk 2]
```diff
@@ -245,30 +217,16 @@ Accept wildcard characters: False
...
-Example: saradavis
+- Distinguished name
+- GUID (objectGUID) 
+- Security identifier (objectSid)
+- SAM account name (sAMAccountName)
 
-This parameter sets the Active Directory attribute with an LDAP Display Name of "managedBy".
-
-The following example shows how to specify this parameter.
+Note: The identifier in parentheses is the LDAP display name for the property.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md:227]:
```suggestion
> [!NOTE]
> The identifier in parentheses is the LDAP display name for the property.
```

File: docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md [hunk 3]
```diff
@@ -592,8 +520,8 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2000KCCISTGSelectionBehaviorEnabled
+Indicates whether the cmdlet to implements the Windows 2000 Server method of ISTG selection.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md:523]:
```suggestion
Indicates whether the cmdlet implements the Windows 2000 Server method of ISTG selection.
```

File: docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md [hunk 4]
```diff
@@ -190,11 +190,11 @@ You can use an instance of an existing site object as a template or you can cons
...
 To retrieve an instance of an existing site object, use the Get-ADReplicationSite cmdlet.
-Then provide this site object to the **Instance** parameter of the New-ADReplicationSite cmdlet to create a new site object.
+Then provide this site object to the *Instance* parameter of the **New-ADReplicationSite** cmdlet to create a new site object.
 You can override property values of the new object by setting the appropriate parameters.
 
 Method 2: Create a new **ADReplicationSite** and set the property values by using the Windows PowerShell command line interface.
-Then pass this object to the **Instance** parameter of the **New-ADReplicationSite** cmdlet to create the new site object.
+Then pass this object to the *Instance* parameter of the **New-ADReplicationSite** cmdlet to create the new site object.
 
 Note: Specified attributes are not validated, so attempting to set attributes that do not exist or cannot be set will raise an error.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md:199]:
```suggestion
> [!NOTE]
> Specified attributes are not validated, so attempting to set attributes that do not exist or cannot be set will raise an error.
```

File: docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md [hunk 5]
```diff
@@ -400,7 +340,7 @@ Accept wildcard characters: False
 ```
 
 ### -ReplicationSchedule
-Default replication schedule for connections within this site (intra-site replication).
+Specifies the default replication schedule for connections within this site (intra-site replication).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md:343]:
```suggestion
Specifies the default replication schedule for connections within this site (intrasite replication).
```

File: docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md [hunk 6]
```diff
@@ -195,25 +178,14 @@ Accept wildcard characters: False
 Specifies an instance of a site object to use as a template for a new site object.
 
 You can use an instance of an existing site object as a template or you can construct a new site object by using the Windows PowerShell command line or by using a script.
-The following examples show how to use these two methods to create a new site object.
 
 Method 1: Use an existing site object as a template for a new object.
 To retrieve an instance of an existing site object, use the Get-ADReplicationSite cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADReplicationSite.md:183]:
```suggestion
To retrieve an instance of an existing site object, use the **Get-ADReplicationSite** cmdlet.
```

File: docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md [hunk 7]
```diff
@@ -98,13 +98,13 @@ Accept wildcard characters: False
 ```
 
 ### -AutomaticInterSiteTopologyGenerationEnabled
-Indicates whether the cmdlet prevents the Kerberos consistency checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
+Indicates whether the cmdlet prevents the Knowledge consistency checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
```
Comment (dariomws) [inline docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md:101]:
```suggestion
Indicates whether the cmdlet prevents the Knowledge Consistency Checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
```

File: docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md [hunk 8]
```diff
@@ -333,13 +335,13 @@ Accept wildcard characters: False
 
 ### -RedundantServerTopologyEnabled
 Indicates whether the cmdlet creates redundant connections between sites before a failure takes place.
-When enabled, disables the Kerberos consistency checker (KCC) failover.
+When enabled, disables the Knowledge consistency checker (KCC) failover.
```
Comment (dariomws) [inline docset/winserver2012r2-ps/addsadministration/New-ADReplicationSite.md:338]:
```suggestion
When enabled, disables the Knowledge Consistency Checker (KCC) failover.
```

**Comment by velkovb** [comment]
Generally there are not examples in the parameter description. I think only AD 2012 articles have it and I do it to have the same parameter description across version. They probably add some value but it is better to be in the examples section.

**Comment by dariomws** [comment]
@velkovb I agree with you, if we only see this in 2012 articles I think it´s a better idea to be consistent and remove them as they are in 2012 R2 and 2016 articles.

**Comment by tseward** [comment]
"[!NOTE]" text should not be part of PowerShell examples as PowerShell (the console) cannot format them. If they are included, "[!NOTE]" and the associated text must be prefixed with ">".

**Comment by e0i** [comment]
> "[!NOTE]" text should not be part of PowerShell examples as PowerShell (the console) cannot format them. If they are included, "[!NOTE]" and the associated text must be prefixed with ">".

Good to go after Trevor's suggestion. cc @velkovb

**Comment by garycentric** [comment]
In review.

**Comment by garycentric** [comment]
@velkovb 

I can't see why the "Note:" items in these articles shouldn't be "> [!NOTE]", and it looks like the instances here only needed to have an angle bracket precede both "[!NOTE]" and the subsequent lines of the note text. 

However, since this repo doesn't have preview on its builds, I'm not going to make changes. I'll merge this PR now.

**Pull request merged**